mne/beamformer/_dics.py

```diff
@@ -117,6 +119,8 @@ def make_dics(info, forward, csd, reg=0.05, noise_csd=None, label=None,
                 The normalization of the weights.
             'src_type' : str
                 Type of source space.
+            'source_nn' : ndarray, shape (n_sources, 3)
+                For each source location, the surface normal.
```

Comment:
Why is this needed for the tutorial? Should this not happen in #10980 instead?

Reply:
This is a good question and point of discussion. It's also related to my suggesting focusing on epochs instead of evoked above. The reasoning behind this is that, as far as I know, it's best coding practice to solve for the most general case, that way you're sure you won't have to do a large rewrite to support a case that you didn't support originally. This would mean using a source estimate resolved by epochs, that is a vector solution and has frequencies as well as times so that you cover all possible use cases (that I can think of). If/when that PR merges, I will refactor it out of this PR but it's nice to have it in here for now so that the tutorial runs top to bottom and we can see the final product!